Objects/frameobject.c

```diff
@@ -310,14 +310,25 @@ framelocalsproxy_dealloc(PyObject *self)
 static PyObject *
 framelocalsproxy_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
```

Comment:
For completeness we could also check that there are no keyword arguments. Currently it accepts and ignores. an arbitrary number of keyword arguments

Reply:
Oh right. I also added a check for keyword arguments.